include/sway/server.h

```diff
@@ -84,6 +88,9 @@ struct sway_server {
 	struct wlr_input_method_manager_v2 *input_method;
 	struct wlr_text_input_manager_v3 *text_input;
 	struct wlr_foreign_toplevel_manager_v1 *foreign_toplevel_manager;
+#if WLR_HAS_XDG_FOREIGN
+	struct wlr_xdg_foreign_registry *foreign_registry;
```

Comment:
Do we need to store this in the server struct? If not, maybe we can just remove this field.

Reply:
Not really, there's no reason to store it here.
Server struct already contains several objects that aren't used anywhere beyond server.c and could be replaced by variables in `server_init` so I assumed that's a part of code style.

Reply:
Yeah, it's mostly cargo culting I think. I'd merge a PR cleaning that up.